solutions/usaco-491.mdx

```diff
@@ -10,6 +10,71 @@ author: Qi Wang
 **Time Complexity:** $\mathcal{O}(N)$
 
 <LanguageSection>
+	
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+unordered_map<int, vector<int>> mp;
+vector<vector<int>> dist;
+
+void bfs(int s, int ix)
```

Comment:
what does s and ix stand for? write it out please!

Reply:
not really addressed
use better variable names instead of just slapping on a comment

Reply:
bump